### ISSUE TITLE
[Supporting the platform - Edit Panel]: Controls present under "Time series graph/Edit panel" blade are partially visible after resizing the page to 200%.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66583]
## User Experience: 
Users with low vision who depend on zooming functionality for better readability will be impacted as the user will not be able to access and identify the controls. 

## Pre-requisite:  
Resize the page to 200%.

## Repro Steps:
Open #Url: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item. Navigate the page "Edit Panel" page which appears. Select "1Time Series graphs" Navigate to the "Override" button and select it.
Navigate the options present under the "Override" pane. Navigate to the "Add Field Override" button and select it. Verify the issue.

## Actual Result: 
Controls present under "Query", "Transform" and "Alert"...etc. present under "Time series graph/Edit panel" blade are not properly visible after resizing the page to 200%.

## Expected Result:
 Controls present under "Query", "Transform" and "Alert"...etc. present under "Time series graph/Edit panel" blade should be clearly visible after resizing the page to 200%.

## Severity
`SEV2`
